src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.controller.js

```diff
@@ -45,6 +45,15 @@
                     $scope.loading = false;
                 }
 
+                $scope.throughput = $scope.endpoint.graphData.throughput;
```

Comment:
Why the second reference? What's wrong with a long binding reference?

`<large-graph plot-points="endpoint.graphData.throughput.points"></large-graph>`